README.md

```diff
@@ -25,117 +25,120 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
    - a controller **interrupt** mechanism, 
    - triggering a **target reset**.
 2. **Firmware modules** that implement the actual functionality of the target device, e.g. controlling stepper and/or servo motors.
-   - Modules exist as separate **include files** (e.g. `ServoI2C_firmware.h`) and are **selected** for compilation via the `firmware_modules.h` file.
+   - Modules exist as separate **include files** (e.g. [`ServoI2C_firmware.h`](firmware/ServoI2C_firmware.h) ) and are **selected** for compilation via the [`firmware_modules.h`](firmware/firmware_modules.h) file.
    - Modules don't have to worry about the I2C overhead but can **concentrate on what's important**: interpreting and reacting to the controller device's commands and requests.
    - Modules can "inject" their code at different places in the firmware, so that there is a **high degree of flexibility**.
 
 The other two basic components are for the **I2C controller's side**:
-3. The **I2Cwrapper class**, provided by the `I2Cwrapper.h` library. 
+
+1. The [**I2Cwrapper class**](https://ftjuh.github.io/I2Cwrapper/class_i2_cwrapper.html), provided by the `I2Cwrapper.h` library. 
    - Controller sketches use an object of type I2Cwrapper to **represent the target** device which handles all low level communication tasks like CRC8 checksums, error handling etc.
    - It also provides **basic functions** for target device management like changing its I2C address, setting an interrupt pin, or making it reset.
-4. **controller libraries** for each module (e.g. `ServoI2C.h`). 
-   - controller libraries use **I2Cwrapper objects** to talk to the target device.
+2. **Controller libraries** for each module (e.g. [`ServoI2C.h`](src/ServoI2C.h)). 
+   - controller libraries use **I2Cwrapper objects** (like the class `ServoI2C` in `ServerI2C.h`) to talk to the target device.
```

Comment:
Doesn't this make it sound as if ServoI2C *is* an I2Cwrapper object? Maybe move the bracket to the end of the sentence?

Reply:
Also typo: "ServerI2C.h"

Reply:
moved, and fixed typo.